L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -508,6 +508,18 @@
                            )
 
 ## GEM-CSC ILT in ME1/1
+from Configuration.Eras.Modifier_run2_GEM_2017_MCTest_cff import run2_GEM_2017_MCTest
+run2_GEM_2017_MCTest.toModify( cscTriggerPrimitiveDigis,
```

Comment:
cscTriggerPrimitiveDigis is used for CSC (clearly, from the names) and this modifier changes its behavior from 2017 MC.
I'm not  sure this one is even supposed to change at all to this logic given VFAT2 used in the slice.

@dildick what is the status of the OTMB firmware for this year: is it aware of GEMs already?
If not, it seems more practical to stay with these mods in cscTriggerPrimitiveDigis only in the run3 era